site/src/components/form.md

```diff
@@ -188,21 +189,102 @@ If you want to skip the basic validation on an input, apply a `data-skip-validat
 		</div>
 	</div>
 	<div class="field">
-		<label class="field-label" for="sample-input">
+		<label class="field-label" for="sample-skip-input">
 			Input
 			<span class="required-indicator"></span>
 		</label>
 		<div class="field-body">
 			<input
-				id="skip-input"
-				name="skip-input"
+				id="sample-skip-input"
+				name="sample-skip-input"
 				class="input"
 				type="text"
 				value=""
+				required
 				data-skip-validation
 			/>
 		</div>
 	</div>
+	<div class="field">
+		<div class="field-body">
+			<label class="checkbox">
+				<input type="checkbox" name="checkbox-example" required id="sample-checkbox" />
+				<span class="checkbox-check" role="presentation" aria-hidden="true"></span>
+				<span class="checkbox-text">
+					Checkbox
+					<span class="required-indicator"></span>
+				</span>
+			</label>
+		</div>
+	</div>
+	<div class="display-flex">
+		<button type="submit" class="button button-primary button-filled">Submit form</button>
+	</div>
+	<div class="submitted-form-data-example"></div>
+</form>
+```
+
+##### Hiding the validation banner
+
+If there is a need to hide the validation banner on top of the form apply a `data-hide-validation-banner` attribute to the `form`.
```

Comment:
If there is a need to hide the validation banner on top of the form, we can apply the `data-hide-validation-banner` attribute to the `form`.

Reply:
Updated